src/pages/CollectionList.tsx

```diff
@@ -12,7 +13,22 @@ function CollectionList() {
 
   return (
     <>
-      <Text as="h1">Collections</Text>
+      <Box display="flex" alignItems="center">
+        <Text as="h1" mr={4}>Collections</Text>
+        <Link 
+          to="/collections/new_collection/"
+          aria-label="Add collection"
+          onClick={e => e.stopPropagation()}
+        >
+          <IconButton
+            aria-label="Add new collection"
+            icon={<Icon as={MdAdd} />}
+            size="md"
+            border="1.5px solid black" // Add black border
+            borderRadius="md" // Optional: Add border radius
```

Comment:
```suggestion
            border="1.5px solid black" // Add black border
            borderRadius="md" // Optional: Add border radius
```

Looks good without the border :)